docs/sidebars.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
       "behaviors/misc",
       "behaviors/hold-tap",
       "behaviors/mod-tap",
+      "behaviors/macros",
```

Comment:
Don't forget to update the status of Macros in `/docs/docs/intro.md` 😃

<img width="631" alt="image" src="https://user-images.githubusercontent.com/60922914/158640187-01ecbb4f-e67c-4446-99a0-78c6e23af633.png">

Reply:
Updated.